### ISSUE TITLE
Bug with some usable items (BB #694)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#694]
This issue was migrated from bitbucket.
**Original Reporter:** PadreWoW
**Original Date:** 08.09.2014 08:38:44 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/694

<hr>

When I tried to use "Totemic Beacon (35828)" I saw "Invalid Target" - but to use this object, the target is not required - only zone "Zoram Strand", but I'm there!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 08.09.2014 08:39:21 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/694/#comment-12216848

<hr>

Screenshot:
![1.jpg](https://bitbucket.org/repo/kMeMB/images/1065766555-1.jpg)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 08.09.2014 08:43:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/694/#comment-12216940

<hr>

The same bug was with the "Netherweave Net"

But this fixes it:

```
#!c++

if ((pItem->GetEntry() == 24268 || pItem->GetEntry() == 24269) && (spellInfo->Id == 31367 || spellInfo->Id == 31368)) // Netherweave net
            {
                uint64 selected = pUser->GetSelection();
                Unit * Target = ObjectAccessor::GetUnit(*pUser, selected);
                if (Target)
                {
                    if (pUser->canAttack(Target))
                    {
                        targets.setUnitTarget(Target);
                    }
                    else{
                        pUser->GetSession()->SendNotification("Invalid Target");
                        continue;
                    }
                }
                else{
                    pUser->GetSession()->SendNotification("No target selected");
                    continue;
                }
            }
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 19.09.2014 20:41:30 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/694/#comment-12453814

<hr>

Implement Spell Dummy Conditions. Fix #694

→ <<cset dadd2163ce6d>>
